src/positions/saga.ts

```diff
@@ -67,4 +88,5 @@ export function* watchFetchBalances() {
 
 export function* positionsSaga() {
   yield spawn(watchFetchBalances)
+  yield spawn(fetchShortcutsSaga)
```

Comment:
I wonder how often we should refetch these. Main problem is if the initial fetch fails.
Maybe for now we could do it everytime the balance needs to be fetched?

Reply:
ah yes i overlooked this, i only was thinking of not trying to fetch too many times. i added a new status to track the fetch in redux, so that we can retry every time the positions are fetched but skip if there was a previous successful request. seemed like not the highest priority work but also if i don't sneak it in now we might never find the motivation to update the logic....